packages/config/src/templates/opStack.ts

```diff
@@ -116,7 +127,7 @@ export function EIGENDA_DA_PROVIDER(
         eigenDACertVersion,
         fallback?.value,
       ),
-      bridge: DA_BRIDGES.NONE,
+      bridge,
       badge: BADGES.DA.EigenDA,
```

Comment:
On the FE the EigenDA badge says "EigenDA with no bridge". As I understand, Celo does use EigenDA with a bridge. So should we create a new badge "EigenDA with a bridge"?